packages/flame/test/game/base_game_test.dart

```diff
@@ -223,4 +223,14 @@ void main() {
     game.update(0.0);
     expect(game.children.isEmpty, equals(true));
   });
+
+  test("can't add a component to a game that don't have layout yet", () {
+    final game = MyGame();
+    final component = MyComponent();
+
+    expect(
+      () => game.add(component),
+      throwsAssertionError,
```

Comment:
should we assert the actual message on each test? since we have 3 different asserts, by just checking this you might not be getting the assert you think you are